src/components/MoviePage/MoviePage.js

```diff
@@ -1,4 +1,5 @@
 import React from "react"
+import PropTypes from "prop-types";
 import "./MoviePage.css"
 
 const MoviePage = (props) => {
```

Comment:
Consider destructuring your props here to avoid the need for `this.props` every time you access a prop.